### PR TITLE
Add prettier to eslint

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: gas_mock/HTTPResponse.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
 class HTTPResponse {
-  constructor (body) {
+  constructor(body) {
```
Comment (kalkov) [inline gas_mock/HTTPResponse.js]: Let's keep the current standard rule : https://eslint.org/docs/rules/space-after-function-name

File: package.json [hunk 2]
```diff
@@ -18,7 +18,11 @@
     "url": "https://github.com/santiment/google-spreadsheet-addon/issues"
   },
   "homepage": "https://github.com/santiment/google-spreadsheet-addon#readme",
-  "dependencies": {},
+  "dependencies": {
+    "eslint-config-prettier": "^6.0.0",
```
Comment (kalkov) [inline package.json]: These should be dev dependencies

File: src/.eslintrc.json [hunk 3]
```diff
@@ -1,17 +1,17 @@
 {
   "root": true,
-  "extends": "eslint-config-es5",
-  "parserOptions": {"ecmaVersion": 5},
-  "plugins": ["googleappsscript"],
+  "extends": ["eslint-config-es5", "prettier"],
+  "parserOptions": {"ecmaVersion": 6},
```
Comment (kalkov) [inline src/.eslintrc.json]: The code in `src` directory will be uploaded to GAS editor w/o transpiling, this is the reason that we have two linters with different rules - one for `src` directory and the other for everything else(runs locally on node.js and won't be uploaded to GAS editor) 

You should keep the ecma version. 

More info on GAS JS version: 
https://developers.google.com/apps-script/guides/services/#basic_javascript_features
https://en.wikipedia.org/wiki/Google_Apps_Script

File: src/api/queries/blockchainQueries.js [hunk 4]
```diff
@@ -1,9 +1,16 @@
-ApiClient_.prototype.fetchActiveAddresses = function (slug, from, to) {
+ApiClient_.prototype.fetchActiveAddresses = function(slug, from, to) {
   var query = {
-    'query': '{\
-       dailyActiveAddresses(slug: "' + slug + '",\
-                            from: "' + toUTC_(from) + '",\
-                            to: "' + toUTC_(to) + '",\
+    query:
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: Looks ugly for prettified code

Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: Hmm, you're right, maybe we should not prettify all files ?

**Comment by kalkov** [comment]
There are a couple of problems with this PR that will take time to resolve and it has low priority, closing it.

**Pull request closed without merging**